client/configuration.cpp

```diff
@@ -103,6 +103,7 @@ configuration::configuration(const std::string & path)
 		                .name = (std::string)i["pretty_name"],
 		                .hostname = (std::string)i["hostname"],
 		                .port = (int)i["port"].get_int64(),
+		                .tcp_only = i["tcp_only"].get_bool(),
```

Comment:
We should accept entires where the bool is not present in case of update from a previous version and server was stored because it had autoconnect property.

Reply:
do we actually want to store this, actually?

the client has no way of knowing if the server is tcp-only, other than from the intent. and the intent being there doesn't mean that the server only works with tcp

Reply:
Right, then don't store it and initialize to false.

Reply:
The address from intent gets stored, it is safer to store tcp_only property as well.

Reply:
updated to `is_bool() && get_bool()`

Reply:
does that work, or do i have to check the key?

Reply:
it should work, that's what we do for other fields